assets/scss/extern-owee.scss

```diff
@@ -510,7 +512,7 @@ body {
 			}
 
 			input {
-				background-color: $donker-rose;
+				background-color: $donkerblauw;
```

Comment:
Ik zou hier gewoon verwijzen naar $primary of $secondary, etc. Maakt de code wat herbruikbaarder